Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/WrapPanel.cs

```diff
@@ -117,16 +117,25 @@ protected override Size MeasureOverride(Size availableSize)
             var parentMeasure = new UvMeasure(Orientation, availableSize.Width, availableSize.Height);
             var spacingMeasure = new UvMeasure(Orientation, HorizontalSpacing, VerticalSpacing);
             var lineMeasure = UvMeasure.Zero;
+            var uIndex = 0;
 
             foreach (var child in Children)
             {
                 child.Measure(availableSize);
-
                 var currentMeasure = new UvMeasure(Orientation, child.DesiredSize.Width, child.DesiredSize.Height);
+                if (currentMeasure.U < 0.1)
```

Comment:
Doesn't this add an unnecessary level of indentation though?

Reply:
I'd rather check for `> 0` than `< .1`. Could be a skinny item. If you want to continue, then change to `== 0`

Reply:
I'd rather check for `> 0` than `< .1`. Could be a skinny item. If you want to continue, then change to `== 0`

Reply:
Sure thing, I had only used `< 0.1` to get rid of the compiler warning about loss of precision (using `==` to compare `double`s) and because I assumed one would probably never have an item smaller than that anyway, but fair point, I'll update this 😊